python/cudf/cudf/core/series.py

```diff
@@ -2391,15 +2391,15 @@ def cov(self, other, min_periods=None):
     @_cudf_nvtx_annotate
     def transpose(self):
         """Return the transpose, which is by definition self."""
-
         return self
 
     T = property(transpose, doc=transpose.__doc__)
 
     @_cudf_nvtx_annotate
     def corr(self, other, method="pearson", min_periods=None):
-        """Calculates the sample correlation between two Series,
-        excluding missing values.
+        """Calculate the sample correlation between two Series.
```

Comment:
Doesn't align with [pandas.Series.corr](https://pandas.pydata.org/docs/reference/api/pandas.Series.corr.html) docs. Of course, this could be intentional if it breaks pydocstyle's rules about being too long on the first line. I'm not sure what to favor in those cases, though I value consistency with pandas pretty highly.

Reply:
I didn't realize this didn't line up with pandas. I also think we should match pandas. E501 (line is too long) is the error that I am most willing to noqa on docstrings, especially when motivated by pandas compat, so I can change this.